draft/implementation-notes/index.html

```diff
@@ -538,6 +538,136 @@ <h2>File Purging</h2>
       </p>
     </section>
 
+    <section>
+      <h2>Migrating to a New Digest Algorithm</h2>
+      <p>
+        Over time new digest algorithms are developed to increase security and address vulnerabilities in existing
```

Comment:
Would it be worth mentioning the extension mechanism for new digest algorithms here?

Reply:
I'm inclined not to because it adds more complication. The spec may evolve or there may be extensions, and for an example we can already use the two that are present.

Reply:
👍